### DISCUSSION TITLE
One file per SC? Or use the Wiki? Or what?

### DISCUSSION THREAD

**Discussion by ShadowBB** [ShadowBB/WCAG_Audit_Examples#3] [General]
Hey people.

I am itching to just start creating a whole bunch of examples. But before I do so I think it might be prudent to have a small discussion on how to organize them. 

Should we use just 1 file per SC and put all the examples there? Possibly categorized by technology (HTML, PDF, Apps, etc).

Or (and I am leaning towards that) should we use 1 file per example and use the Github Wiki functionality to actually categorize the examples on SC pass/failure/unclear and have different Wiki pages per SC per technology?

Or does anybody else have a different thought?

**Comment by sander-nl** [comment]
I don't have experience with GitHub wikis, but I see them merely as resource material. 
I'm feeling for one file (html or markdown) per sc with the actual decisions. Discussions can live here (under "Discussions") or as issues.
